menu_generator.rb

```diff
@@ -0,0 +1,17 @@
+puts "Welcome to the Kreepy Kavern! \nour menu today is:\n "
+
+texture = ["Gellatinous", "Spongy", "Semi-solid", "Pulpy", "Crystallized",
+              "Frozen", "Condensed", "Liquid", "Loose", "Hot", "Fleshy",
+              "Plump", "Chunky", "Soupy", "Putrid"]
+descriptor = ["Herbaceous", "Sludgy", "Savory", "Pungent", "Zesty",
+              "Frothy", "Fermented", "Dull", "Stale", "Watery", "Boiled",
+              "Melted", "Sour", "Wad of", "Block of"]
+food = ["Cereal", "Spam", "Beef Tartare", "Pig's Feet", "Tofu",
+              "Mung Beans", "Cheese", "Hot Pocket", "Lean Cuisine",
+              "Bologne", "Mayonnaise", "Saurkraut", ]
+
+menu = 1
+while menu <= texture.length
```

Comment:
A while loop is a great choice here, but you could also use an `times` loop here, which comes with a handy enumerator built in:

```ruby
texture.length.times do |menu|
```